ae/server/api_routes.py

```diff
@@ -38,6 +38,7 @@
 
 class CommandQueryModel(BaseModel):
     command: str = Field(..., description="The command related to web navigation to execute.")  # Required field with description
+    llm_config: str | None = Field(None, description="The LLM configuration string to use for the agents.")
```

Comment:
Why are we using a string rather than a `dict[str,Any]`? The input to the api is already in JSON format, why not keep up with that?
Also we should have this documented with samples that show up in the endpoint docs and readme for people to follow it